packages/adapter-node/package.json

```diff
@@ -45,10 +45,11 @@
 		"@polka/url": "^1.0.0-next.28",
 		"@sveltejs/kit": "workspace:^",
 		"@sveltejs/vite-plugin-svelte": "^5.0.1",
-		"@types/node": "^18.19.48",
+		"@types/node": "^20.17.16",
+		"crossws": "^0.3.3",
 		"polka": "^1.0.0-next.28",
 		"sirv": "^3.0.0",
-		"typescript": "^5.3.3",
+		"typescript": "^5.7.3",
```

Comment:
typescript was bumped to 5.7 because that's what unjs/crossws [is using](https://github.com/unjs/crossws/blob/bc55c9765f436316213e9a3b907522cc86013a8c/package.json#L110) and it uses a generic version of UInt8Array that's [only available in version 5.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-7.html#support-for---target-es2024-and---lib-es2024)